modules/ROOT/partials/konflux-key-features.adoc

```diff
@@ -1,27 +1,46 @@
-* *Robust Security*
+=== CI/CD Automation
 
-+
-Enhance the security of your software supply chain. {ProductName} offers unparalleled transparency into your software’s composition, helping you address vulnerabilities before they become issues.
+{ProductName} simplifies your DevOps processes through:
 
-* *Customizable Compliance*
+* *Automated Pipelines*: {ProductName} provides default pipeline configurations for building, testing, and deploying production software. You can trigger these pipelines via Git events, such as when team members accept push or pull requests. {ProductName} posts the results back to your source control management system.
 
-+
-Achieve the highest security standards and customize your development process to meet specific project requirements. {ProductName}'s adaptable platform ensures compliance without sacrificing flexibility.
+* *Integration with GitHub and GitLab*: {ProductName} allows you to integrate with Tekton. This integration enables you to run Tekton PipelineRuns based on GitHub or GitLab events, which you can define in your repository's `.tekton` folder. We will add more source control management providers in the future.
```

Comment:
I personally think saying that Konflux allows you to integrate with Tekton sounds a little off. We're integrating with Github/Gitlab, and Tekton is the framework used to run the pipelines in the Konflux cluster.

Reply:
I may still be a little fuzzy on this, but I think I got it right in the document. Please let me know if any further changes are needed.